README.md

```diff
@@ -1,21 +1,80 @@
 # Auth0 Product Education Labs
 
+A set of tools and example applications for Product Education courses.
+
+#### Running Locally
+
+All of the tools and sample applications can be run locally assuming dependecies are installed on the local machine. To prepare for local execution run `npm run init` from the root of the repository.
```

Comment:
Not clear that I needed to do an `npm install` first

Reply:
I have clarified this in the readme of #2. See [here](https://github.com/auth0/auth0-product-education-labs/blob/vercel-expense-api-unsecured/README.md#running-locally). This means that this update will go into master once both PRs are accepted.